HelpSource/Classes/Array.schelp

```diff
@@ -57,37 +57,47 @@ copymethod:: ArrayedCollection *series
 
 copymethod:: ArrayedCollection *iota
 
-copymethod:: SequenceableCollection *interpolation
+copymethod:: Collection *interpolation
 
-copymethod:: SequenceableCollection *rand
-
-copymethod:: SequenceableCollection *rand2
-
-copymethod:: SequenceableCollection *linrand
-
-copymethod:: SequenceableCollection *exprand
-
-copymethod:: SequenceableCollection *fib
+copymethod:: Collection *rand
 
+copymethod:: Collection *rand2
 
+copymethod:: Collection *linrand
 
+copymethod:: Collection *exprand
 
+copymethod:: Collection *fib
```

Comment:
that makes sense. probably asking help to someone to do that would be a great idea, as I can't quite figure out how to do that on my own